x-pack/plugins/observability/public/components/custom_threshold/components/alert_details_app_section/alert_details_app_section.tsx

```diff
@@ -85,6 +87,36 @@ export default function AlertDetailsAppSection({
   const alertEnd = alert.fields[ALERT_END] ? moment(alert.fields[ALERT_END]).valueOf() : undefined;
   const groups = alert.fields[ALERT_GROUP];
   const tags = alert.fields[TAGS];
+  const CHART_TITLE_LIMIT = 120;
+
+  const generateChartTitle = (criterion: MetricExpression) => {
+    let equationTitle = '';
+    const symbolResolver: Record<string, string> = {};
```

Comment:
nit: maybe `metricNameResolver`

Reply:
updated in edca4c626a79f713e53ac714fedfffa97cac8b48